src/ubuntu/manifest.json

```diff
@@ -334,7 +352,7 @@
           "platforms": [
             {
               "architecture": "amd64",
-              "dockerfile": "src/ubuntu/22.04/helix/amd64",
+              "dockerfile": "src/ubuntu/22.04/helix/any",
```

Comment:
While I'm indifferent to the use of the `any` directory pattern in general, it doesn't align with the existing pattern in this repo for multi-arch Dockerfiles. For example, https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/ubuntu/20.04/Dockerfile is a multi-arch Dockerfile that is placed at the "root". In the context of this PR, that would mean placing the Dockerfile at `src/ubuntu/22.04/helix`.

Reply:
Yes please

Reply:
Done